deployment/v3/external/postgres/generate-secret.py

```diff
@@ -38,7 +38,7 @@
   namespace: postgres
 type: Opaque
 data:
-  postgresql-password: {postgres_base64_password}
+  postgres-password: {postgres_base64_password}
```

Comment:
is it encrypted base64 ?

Reply:
It is Base64 encoding where the password changes into base64 format and is stored as a secret, it isn't a form of encryption.